master_changes.md

```diff
@@ -82,6 +82,9 @@ New option/command/subcommand are prefixed with ◈.
   * Add E67 checksum specified with non archive url [#4635 @rjbou]
   * Disable subpath warning E63,W64 [#4638 @rjbou]
 
+## Repository
+  * When several checksume are specified, instead of creating add in the cache only the first checksum, add best one and link others [#XXX rjbou]
```

Comment:
You should specify that this concerns `opam admin`

Reply:
It is for admin and for regular cache also (called from OpamRepository.pull_from_upstream)